frontend/components/commons/results/ResultsList.vue

```diff
@@ -16,32 +16,36 @@
   -->
 
 <template>
-  <div class="list">
-    <slot name="header" />
+  <div class="content">  
+    <slot name="header" />sssssdhuiehdiuehwudi
     <div class="results-scroll" id="scroll">
       <div :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }" v-if="showLoader">
         <results-loading :size="dataset.viewSettings.pagination.size" />
       </div>
       <DynamicScroller
-        v-else
         page-mode
         class="scroller"
         :items="visibleRecords"
         :min-item-size="150"
         :buffer="200"
         :style="{ paddingTop: `${dataset.viewSettings.headerHeight + 10}px` }"
       > 
+        <template #before>
+          <slot name="results-header"/>
+          <results-empty :title="emptySearchInfo.title" :message="emptySearchInfo.message" :icon="emptySearchInfo.icon" v-if="dataset.results.total === 0" />
+        </template>
```

Comment:
Why we cannot use the `header` slot to handle this info ?

Reply:
Because `header` slot  is outside the scroll area and we want it inside